helm/kubernetes-nginx-ingress-controller-chart/charts/http-backend-service-type-migration/templates/np.yaml

```diff
@@ -0,0 +1,24 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: {{ .Values.name }}-allow
```

Comment:
It has same name/namespace than np before and following. there will be a collision?

Reply:
Migrations have their own names

Reply:
fair point 🤦‍♂